generator/templates/generator/_make_release_public.md

```diff
@@ -1,9 +1,21 @@
-{% load generator_extras %}
-- [ ] Ensure separated folder in origin.djangoproject.com for the release (this should be the case already)
-  - `ssh www@origin.djangoproject.com 'ls -l /home/www/www/media/releases/{{ release.feature_version }}'`
-- [ ] **ONLY SCP** each binary set (tar.gz and wheel) to the corresponding folder (use commands from before)
+- [ ] Go to the [Add release page in the admin](https://www.djangoproject.com/admin/releases/release/add/), enter the following:
+  - new release number
+  - Version: {{ release }}
+  - Is active: False
+  - LTS: {{ release.is_lts }}
+  - Release date: release.when
```

Comment:
```suggestion
  - Release date: {{ release.when }}
```
maybe

Reply:
Looks like a bug and that a test is missing :-) Thank you! :star2: